tests/CMakeLists.txt

```diff
@@ -160,7 +160,7 @@ if (PY_MYPY)
      )
 endif()
 
-if (PY_OPENPYXL AND PY_PANDAS AND PYTHON3_EXECUTABLE)
+if (PYTHON_VERSION_MAJOR GREATER 2 AND PY_OPENPYXL AND PY_PANDAS AND SRG_PRODUCT_RHEL9)
```

Comment:
isn't it `SSG_PRODUCT_RHEL9` instead of `SRG...`?

Reply:
great, thanks for finding this, it should be SSG_PRODUCT_RHEL9

@Mab879 PTAL

Reply:
https://github.com/ComplianceAsCode/content/pull/9964 should fix that.